esp-hal/src/dma/pdma.rs

```diff
@@ -90,6 +90,11 @@ impl<C: PdmaChannel<RegisterBlock = SpiRegisterBlock>> RegisterAccess for SpiDma
 }
 
 impl<C: PdmaChannel<RegisterBlock = SpiRegisterBlock>> TxRegisterAccess for SpiDmaTxChannelImpl<C> {
+    fn set_auto_write_back(&self, enable: bool) {
+        // there is no `auto_wrback` for SPI
+        core::assert!(!enable);
```

Comment:
This means the SPI_DMA doesn't clear the owner bit right?

Reply:
At least I haven't found a way to enable it

Reply:
probably just a habit to write it like that ... need to unlearn that 😄